tests/x509/test_x509.py

```diff
@@ -734,6 +734,35 @@ def test_load_multiple_sections(self, backend):
         )
         assert cert == cert2
 
+    @pytest.mark.supported(
+        only_if=lambda backend: backend.x509_pss_supported(),
+        skip_message="Requires OpenSSL with X.509 PSS RSA support (1.1.1+)",
+    )
+    def test_load_pss_cert(self, backend):
+        cert = _load_cert(
+            os.path.join("x509", "custom", "rsa_pss.pem"),
+            x509.load_pem_x509_certificate,
+            backend,
+        )
+        assert cert.signature_algorithm_oid == SignatureAlgorithmOID.RSASSA_PSS
+        public_key = cert.public_key()
+        assert isinstance(public_key, rsa.RSAPublicKey)
+        with pytest.raises(UnsupportedAlgorithm):
+            cert.signature_hash_algorithm
```

Comment:
There's no way here to actually get the PSS parameters, which would be required to actually verify the signature, is there?

Reply:
If we want to get them we'll need to parse this object ourselves (and also figure out an API for providing them, seems like overloading signature_hash_algorithm is probably not a good idea)